ckanext/switzerland/plugin.py

```diff
@@ -136,10 +142,32 @@ def get_helpers(self):
         }
 
 
+class FormatMappingNotLoadedException(Exception):
```

Comment:
Just some nitty-gritty: if you define your own exceptions, they should end with "Error" instead of "Exception" (https://docs.python.org/3/tutorial/errors.html#user-defined-exceptions). Please rename your own exception to `FormatMappingNotLoadedError`.